doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,72 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization. The HoloViz Organization is an open-source and collaborative effort with a mission to develop solutions for a wide range of visualization problems in Python.
+
+## 1. Terminology
+- "HoloViz Organization" or "Organization": The entire HoloViz ecosystem of Contributors and Projects
+- "HoloViz GitHub Organization": The Github organization at https://github.com/holoviz
+- "HoloViz" repo: The specific repository at https://github.com/holoviz/holoviz
+- "HoloViz.org": The website at https://holoviz.org/
+
+## 2. Organization Steering Committee
+By adding their name to the [Steering Committee.md file](./STEERING-COMMITTEE.md), Steering Committee members agree to the policies in this document.
```

Comment:
Unclear about the approval process for new members, do the regular voting rules apply or is there a distinct process?